test/UID2.Client.Test/UID2.Client.Test.csproj

```diff
@@ -6,6 +6,14 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;618</NoWarn>
```

Comment:
Where did these come from?

Reply:
I added 618 because the tests are intentionally calling lots of obsolete methods. 1701;1702 seem to be defaults